incubator/consul/templates/consul-petset.yaml

```diff
@@ -157,6 +157,12 @@ spec:
       - name: gossip-key
         secret:
           secretName: gossip-key
+      readinessProbe:
+        httpGet:
+          path: /v1/health/service/consul
```

Comment:
Dumb question.  Does this work with clustered consul?  With the master when it was in boostrap mode, we had issues with the readinessProbe working.